ci-operator/config/openshift/release/openshift-release-master__ci-4.7-upgrade-from-stable-4.6.yaml

```diff
@@ -110,6 +110,17 @@ tests:
       TEST_TYPE: upgrade
       TEST_UPGRADE_OPTIONS: abort-at=99
     workflow: openshift-upgrade-aws
+- as: e2e-aws-umw-upgrade
```

Comment:
You don't need upgrade in the name, everything in this file is an upgrade.  `e2e-aws-umw`

Reply:
I'd prefer to have upgrade to be in the prowjob name, without it folks may assume its an install job

Reply:
Won't the prow job get called `ci-openshift-release-master-ci-4.8-upgrade-from-stable-4.7-e2e-aws-umw-upgrade`, i.e. upgrade appears twice?

Reply:
Ah, okay, sure. Renameing this

Reply:
Disregard, I guess the other jobs in here have the same suffix